includes/validation/class-amp-validation-manager.php

```diff
@@ -2114,6 +2114,17 @@ public static function validate_url( $url ) {
 			$validation_url
 		);
 
+		// Strip byte order mark (BOM).
+		if ( "\xEF\xBB\xBF" === substr( $response, 0, 3 ) ) {
```

Comment:
I would have turned this into a `while` loop, as under certain circumstances, some systems prepend the BO without checking if there is one already. So you can end up with multiple ones as well.

Reply:
Good point. Done in #4683.